packages/server-wallet/jest/jest.with-peers.config.js

```diff
@@ -1,3 +1,5 @@
 const config = require('./jest.config');
 config.testMatch = ['<rootDir>/src/**/__test-with-peers__/**/?(*.)test.ts'];
+// We don't want to run ./jest/knex-setup-teardown.ts' as it assumes a database called SERVER_WALLET
```

Comment:
This is according to the PR description:
```suggestion
// We don't want to run ./jest/knex-setup-teardown.ts' as it assumes a database called server_wallet_test
```